packages/data-api/scripts/buildAnalyticsMaterializedView.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
I always used `set -e` as a separate command - didn't know you can also use it this way, nice!